frontend/components/utils/BidInput.vue

```diff
@@ -62,10 +62,8 @@ export default Vue.extend({
                 return;
             }
             const bidTopLimit = maxValue?.minus(minValue);
-            if (currentValue?.isGreaterThan(bidTopLimit) || currentValue?.isLessThan(minValue)) {
-                throw new Error(
-                    `The value can only be between ${minValue.toFixed(2)} and ${bidTopLimit.toFixed(2)} or the maximum`
-                );
+            if (currentValue?.isGreaterThan(bidTopLimit)) {
+                throw new Error(`The value can only be between 0 and ${bidTopLimit.toFixed(2)} or the maximum`);
             }
             if (maxValue?.isLessThan(minValue.multipliedBy(2))) {
                 throw new Error('The value can not be changed since the leftover part will be too small');
```

Comment:
I assume this one needs adjustment as well since we also apply this multiplication by 2

Otherwise I am running into the problem indicated below:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/36229155/167682920-bfd539cc-f7ee-44f3-9fc8-3e67a8c1bdd3.png">

<img width="500" alt="image" src="https://user-images.githubusercontent.com/36229155/167682982-08648b82-6ac3-44b1-819d-41396b6f827e.png">